From 608f2e62f1ca6d526bc47f75452b362494b4733b Mon Sep 17 00:00:00 2001
From: Hieu Huynh <hieuh@os.amperecomputing.com>
Date: Mon, 26 Feb 2024 08:11:36 +0000
Subject: [PATCH] Add timeout when setting NTP mode

Commit ecd45a8 introduces a timeout when NTP mode is set to 20
seconds. 20s is not enough. To resolve the problem, this raises
the timeout to 60 seconds.

Change-Id: If060917cf03a78185235926dbf11c2f2e0307268
Signed-off-by: Hieu Huynh <hieuh@os.amperecomputing.com>
---
 src/store/modules/Settings/DateTimeStore.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/store/modules/Settings/DateTimeStore.js b/src/store/modules/Settings/DateTimeStore.js
index 51b722a..adfed54 100644
--- a/src/store/modules/Settings/DateTimeStore.js
+++ b/src/store/modules/Settings/DateTimeStore.js
@@ -56,7 +56,7 @@ const DateTimeStore = {
              * TODO: remove timeout if backend solves
              * https://github.com/openbmc/openbmc/issues/3459
              */
-            const timeoutVal = state.isNtpProtocolEnabled ? 20000 : 0;
+            const timeoutVal = state.isNtpProtocolEnabled ? 60000 : 0;
             return await new Promise((resolve, reject) => {
               setTimeout(() => {
                 return api
-- 
2.25.1

